materials/sections/metadata-data-best-practices-intro.Rmd

```diff
@@ -175,21 +175,15 @@ Researchers should get in the habit of citing the data that they use - even if i
 
 We recently implemented this “Register Citation” feature to allow researchers to register known citations to their datasets. Researchers may register a citation for any occasions where they know a certain publication uses or refers to a certain dataset, and the citation will be viewable on the dataset profile within 24 hours. 
 
-To register a citation, navigate to the dataset using the DOI and click on the citations tab. Once there, this dialog box will pop up and you'll be able to register the citation with us.
+To register a citation, navigate to the dataset using the DOI and click on the citations tab. Once there, this dialog box will pop up and you'll be able to register the citation with us. Click that button and you'll see a very simple form asking for the DOI of the paper and if the paper **CITES** the dataset (that is, the dataset is explicitly mentioned somewhere in the text) or **USES** the dataset (that is, uses the dataset but doesn't formally cite it).
```

Comment:
Rather than "explicitly mentioned somewhere in the text", how about use "is explicitly identified or linked to somewhere in the text or references".